cmd2/cmd2.py

```diff
@@ -1884,17 +1892,22 @@ def _perform_completion(
             return
 
         # Determine the completer function to use
-        if command is not None:
+        if custom_settings is None:
```

Comment:
Why was this logic changed?

Reply:
The only way for `command` to not be `None` is if `custom_settings` is `None`. The new logic is redundant.

Reply:
As with all changes here, to make the appease the type validator.

Reply:
I'd like to refactor this code or something. The new solution ends up with a redundant block that never gets unit tested since it can't actually occur.

```Python
if command is None:
    return
```

Reply:
What was the type validator error on this code?

Reply:
I put a cast where it was complaining instead.